### PR TITLE
Seedless Procrustes reference added

### LINKED ISSUE #645: [BUG] Seedless Procrustes reference 1 is just a TODO in the docs
see here: https://github.com/microsoft/graspologic/blob/dev/graspologic/align/seedless_procrustes.py#L108

### PR DESCRIPTION
**Pull request by rajpratyush**
#### Reference Issues/PRs
closes #645 

#### What does this implement/fix? Briefly explain your changes.
Added the Seedless Procrustes reference and removed the comment

### CODE REVIEW

File: docs/reference/reference/utils.rst [hunk 1]
```diff
@@ -8,7 +8,7 @@ Transformations
 
 .. autofunction:: pass_to_ranks
```
Comment (loftusa) [inline docs/reference/reference/utils.rst:9]: ?

File: docs/reference/release/release_0.0.2.rst [hunk 2]
```diff
@@ -34,7 +34,7 @@ API Changes
 -----------
 - ``check_lcc`` argument in ``AdjacencySpectralEmbed``, ``LaplacianSpectralEmbed``, and ``OmnibusEmbed`` classes, which checks if input graph(s) are fully connected when ``check_lcc`` is True.
 - ``gridplot`` and ``heatmap`` now have a ``inner_hier_labels`` and ``outer_hier_labels``, which are used for hierarchical labeling of nodes.
-- ``to_laplace`` function now has ``regularizer`` arg for when ``form`` is 'R-DAD'.
+- ``to_laplacian`` function now has ``regularizer`` arg for when ``form`` is 'R-DAD'.
```
Comment (loftusa) [inline docs/reference/release/release_0.0.2.rst:37]: this PR shouldn't have to update all this stuff

**Review by loftusa — CHANGES_REQUESTED** [review]
@rajpratyush Lots of extra changes besides the reference - did you merge with upstream first? Would like this PR to only change the reference before merging

File: docs/reference/release/release_0.0.2.rst (hunk 2 above)
Comment (rajpratyush) [reply docs/reference/release/release_0.0.2.rst:37]: lemme create a new pr

**Pull request closed without merging**